### ISSUE TITLE
Error converting chat message to Text in MessageEvent

### ISSUE THREAD

**Issue by zml2008** [SpongePowered/Sponge#213]
getMessage() in the chat event doesn't work. care to take a look @Minecrell (or @gratimax ?)

```
[22:58:40] [Server thread/WARN] [Sponge]: A handler raised an error when handling an event
java.lang.NullPointerException: translation
        at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:229) ~[minecraft_server.1.8.jar:?]
        at org.spongepowered.api.text.TextBuilder$Translatable.translation(TextBuilder.java:636) ~[TextBuilder$Translatable.class:1.8-1334-1.1DEV-0+unknown-b0.git-unknown]
        at org.spongepowered.api.text.TextBuilder$Translatable.<init>(TextBuilder.java:553) ~[TextBuilder$Translatable.class:1.8-1334-1.1DEV-0+unknown-b0.git-unknown]
        at org.spongepowered.api.text.Texts.builder(Texts.java:217) ~[Texts.class:1.8-1334-1.1DEV-0+unknown-b0.git-unknown]
        at net.minecraft.util.ChatComponentTranslation.createBuilder(SourceFile:44) ~[hz.class:?]
        at net.minecraft.util.ChatComponentStyle.toText(SourceFile:108) ~[hj.class:?]
        at net.minecraftforge.event.ServerChatEvent.getMessage(ServerChatEvent.java:59) ~[ServerChatEvent.class:?]
        at com.zachsthings.testplugin.TestPlugin.onChat(TestPlugin.java:37) ~[TestPlugin.class:?]
        at org.spongepowered.mod.event.handler.MessageEventHandler_TestPlugin_onChat3.handle(Unknown Source) ~[?:?]
        at org.spongepowered.mod.event.SpongeEventBus.callListener(SpongeEventBus.java:209) [SpongeEventBus.class:1.8-1334-1.1DEV-0+unknown-b0.git-unknown]
        at org.spongepowered.mod.event.SpongeEventBus.post(SpongeEventBus.java:269) [SpongeEventBus.class:1.8-1334-1.1DEV-0+unknown-b0.git-unknown]
        at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:56) [EventBus.class:?]
        at net.minecraftforge.common.ForgeHooks.onServerChatEvent(ForgeHooks.java:431) [ForgeHooks.class:?]
        at net.minecraft.network.NetHandlerPlayServer.func_147354_a(NetHandlerPlayServer.java:796) [rj.class:?]
        at net.minecraft.network.play.client.C01PacketChatMessage.func_180757_a(SourceFile:37) [lu.class:?]
        at net.minecraft.network.play.client.C01PacketChatMessage.func_148833_a(SourceFile:9) [lu.class:?]
        at net.minecraft.network.PacketThreadUtil$1.run(SourceFile:13) [ih.class:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_25]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_25]
        at net.minecraftforge.fml.common.FMLCommonHandler.callFuture(FMLCommonHandler.java:709) [FMLCommonHandler.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:655) [MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:364) [po.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:598) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:478) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_25]
[22:58:40] [Server thread/INFO]: <zml> oh noes
[22:58:52] [Server thread/INFO]: zml lost connection: TextComponent{text='Disconnected', siblings=[], style=Style{hasParent=false, color=null, bold=null, italic=null, underlined=null, obfuscated=null, clickEvent=null, hoverEvent=null, insertion=null}}
```

**Comment by gratimax** [comment]
[Found it](https://github.com/SpongePowered/Sponge/blob/c5022641676a1e2b76225966473b0802f8f85dfd/src/main/java/org/spongepowered/mod/mixin/core/text/MixinChatComponentTranslation.java#L44)

**Comment by gratimax** [comment]
[Found it](https://github.com/SpongePowered/Sponge/blob/c5022641676a1e2b76225966473b0802f8f85dfd/src/main/java/org/spongepowered/mod/mixin/core/text/MixinChatComponentTranslation.java#L44)

**Comment by zml2008** [comment]
thanks :)

**Comment by Minecrell** [comment]
I haven't/hadn't implemented the Translation API for some reason yet, so I put in null in there for now

**Comment by Minecrell** [comment]
Should be fixed in b4570ba7162552bacf8457e727bc08e5666f0ade
